application/actions/meetings_test.go

```diff
@@ -170,3 +173,80 @@ func (as *ActionSuite) Test_RecentMeetingsQuery() {
 	as.Equal(wantMtg.EndDate.Format(domain.DateFormat), gotMtg.EndDate,
 		"incorrect meeting EndDate")
 }
+
+func (as *ActionSuite) Test_CreateMeeting() {
+	f := createFixturesForMeetings(as)
+	user := f.Users[0]
+
+	input := `imageFileID: "` + f.File.UUID.String() + `"` +
+		`
+			name: "name"
+			description: "new description"
+			location: {description:"meeting location" country:"dc" latitude:1.1 longitude:2.2}
+			startDate: "2025-03-01"
+			endDate: "2025-03-21"
+			moreInfoURL: "example.com"
+		`
+	query := `mutation { meeting: createMeeting(input: {` + input + `}) 
+			{ createdBy { nickname } imageFile { id } name
+			description location { description country latitude longitude }
+			startDate endDate moreInfoURL }}`
+
+	var resp meetingResponse
+	as.NoError(as.testGqlQuery(query, user.Nickname, &resp))
+
+	gotMtg := resp.Meeting
+
+	emptyUUID := uuid.UUID{}
+	as.NotEqual(emptyUUID, gotMtg.ID, "don't want empty UUID")
```

Comment:
I think this isn't testing correctly because they are different types. That's why I don't like `NotEqual`.
```suggestion
	as.True(emptyUUID.String() != gotMtg.ID, "don't want empty UUID")
```

Reply:
Good catch.